configs/mainnet.json

```diff
@@ -483,5 +531,119 @@
+      "decimals": 6
+    },
+    "Cosmos Hub": {
+      "denom": "ibc/A4DB47A9D3CF9A068D454513891B526702455D3EF08FB9EB558C561F9DC2B701",
+      "decimals": 6
+    },
+    "Kujira": {
+      "denom": "ibc/662914D0C1CEBCB070C68F061D035E8B10A07C79AB286E7342C85F3BE74612C5",
+      "decimals": 6
+    },
+    "Noble": {
+      "denom": "usdc.1.6.53ff75",
+      "decimals": 6
+    },
+    "Stargaze": {
+      "denom": "ibc/C688EBE80ECC20865331A18D8D19654754F6B189977715FED00FBA2BE41F144F",
+      "decimals": 6
+    },
+    "Stride": {
+      "denom": "ibc/3552CECB7BCE1891DB6070D37EC6E954C972B1400141308FCD85FD148BD06DE5",
+      "decimals": 18
+    },
+    "Terra": {
+      "denom": "ibc/2B58B8C147E8718EECCB3713271DF46DEE8A3A00A27242628604E31C2F370EF5",
+      "decimals": 6
+    },
+    "Mantle": {
+      "denom": "bit.1.2.9d68c4",
+      "decimals": 18
+    },
+    "Base": {
+      "denom": "brdg/1299d2e94c483e72f64100088886b8a5b7c13595047e53e032c9938d07a98c20",
+      "decimals": 18
+    },
+    "Evmos": {
+      "denom": "ibc/35E771B8682D828173F4B795F6C307780F96DC64D6F914FAE4CC9B4666F66364",
+      "decimals": 6
+    },
+    "Juno": {
+      "denom": "ibc/9E3EB38E5E157AEBFF4A8EAC66E654BC8ECFCB1F758F4D1C0F2D65945E9E2935",
+      "decimals": 6
+    },
+    "Comdex": {
+      "denom": "ibc/5E60F861861B36902155FCADB0B3AD1D474376A6D4CFDBE1FC60E262016CEF7A",
+      "decimals": 6
+    },
+    "Stafihub": {
+      "denom": "ibc/0249D3BAB5191A1CB96C66F76DD5B54BBBE8A8391C38691C4B980E82F385086B",
+      "decimals": 6
+    },
+    "Persistence Core": {
+      "denom": "ibc/AAA241B0C1B237AA252F939C7CB7AB4C21A46BAA97464DDB8B5C92D4AF635E29",
+      "decimals": 6
+    },
+    "Canto": {
+      "denom": "ibc/92E974290AF9E2BC3AEEEC35305C8FD76AC5A22A74CF8D91270FDF5A1C41E861",
+      "decimals": 18
+    },
+    "Sommelier": {
+      "denom": "ibc/96BB9743B631E66525F13E5E944AF1A222AF928A4C870E457CDE932092F9528B",
+      "decimals": 18
+    },
+    "IRIS Hub": {
+      "denom": "ibc/4E06CF24FEBFB3F5AF645377DCC0B70AA6183BAF6B918B8B6243FCDEB7D38118",
+      "decimals": 6
+    },
+    "Ethereum": {
+      "denom": "brdg/e3877e2c8ed05e404e04417747e2f430285c11dc839663b943b39ff7cbb56377",
+      "decimals": 18
+    },
+    "Arbitrum": {
+      "denom": "brdg/e3877e2c8ed05e404e04417747e2f430285c11dc839663b943b39ff7cbb56377",
+      "decimals": 18
+    },
+    "Polygon": {
+      "denom": "brdg/e2238fb8fc5f08299c1789177d6d89bb2853238a65f6e48a9ec3b67e289f5601",
+      "decimals": 18
+    },
+    "Binance Smart Chain": {
+      "denom": "brdg/1768794901f8a19c2ec795a5402653cef6cbfe6b3ec6398d39fc37de963cb667",
+      "decimals": 18
+    },
+    "Neo": {
+      "denom": "bneo.1.14.e2e5f6",
+      "decimals": 8
+    },
+    "Neo3": {
+      "denom": "bneo.1.14.e2e5f6",
+      "decimals": 8
+    },
+    "Optimism": {
+      "denom": "brdg/e3877e2c8ed05e404e04417747e2f430285c11dc839663b943b39ff7cbb56377",
```

Comment:
For Optimism, denom should be `brdg/34a330df380acf4845e3c3341722a73cc62b57819a2e9960fbb0e040a5210423`